### PR TITLE
Fix minor bugs and clean up code.

### PR DESCRIPTION
**Pull request by anwar6953**
http://builds.cask.co/browse/CDAP-DUT4730-1

### CODE REVIEW

File: cdap-client/src/main/java/co/cask/cdap/client/MetricsClient.java [hunk 1]
```diff
@@ -160,13 +160,14 @@ public MetricQueryResult query(Map<String, String> tags, List<String> metrics, L
...
    * @param metrics names of the metrics
    * @param groupBys groupBys for the request
+   * @param timeRangeParams parameters specifying the time range
    * @return values of the metrics
    * @throws IOException if a network error occurred
    * @throws UnauthenticatedException if the request is not authorized successfully in the gateway server
    */
   // TODO: take in query object shared by MetricsHandler
   public MetricQueryResult query(Map<String, String> tags, List<String> metrics, List<String> groupBys,
-                                 @Nullable Map<String, String> timeRangeParams)
```
Comment (gokulavasan) [inline cdap-client/src/main/java/co/cask/cdap/client/MetricsClient.java:169]: Why is nullable annotation being removed? We don't support that in this method?

**Comment by gokulavasan** [comment]
One question. Rest LGTM 👍

File: cdap-client/src/main/java/co/cask/cdap/client/MetricsClient.java (hunk 1 above)
Comment (anwar6953) [reply cdap-client/src/main/java/co/cask/cdap/client/MetricsClient.java:169]: the method doesn't actually handle the null case.
null is actually never passed in either, for the current usages.

Comment (gokulavasan) [reply cdap-client/src/main/java/co/cask/cdap/client/MetricsClient.java:169]: ok.

**Pull request merged**